src/shared/Git-Credential-Manager/Program.cs

```diff
@@ -76,13 +76,6 @@ private static void AppMain(object o)
                     {
                         appPath = appPath.Substring(0, appPath.Length - 4);
                     }
```

Comment:
We can also delete the containing block and commentary above.

Reply:
Oops. That's so clear in Rider, but was less so in VS Code.